uhu/updatehub/http.py

```diff
@@ -6,10 +6,18 @@
 from ._request import Request, HTTPError
 
 
+from ..utils import get_custom_ca_certs_file
```

Comment:
By convention, this import must be before import from line 6 with no line separation. The idea behind is to left the nearest imports by the end. Also, since these two imports come from the same package, there is no need for line separation.